forms-flow-admin/src/components/roles/roles.tsx

```diff
@@ -358,14 +359,15 @@ const Roles = React.memo((props: any) => {
         </Modal.Body>
         <Modal.Footer>
           <button type="button"
-            className="btn btn-link text-dark" onClick={handleCloseEditRoleModal}>
+            className="btn btn-link text-dark" onClick={handleCloseEditRoleModal}  data-testid="edit-role-modal-cancel-button">
```

Comment:
here you used button but above used btn end of the data testid so make them similar

Reply:
`button `is you used more

Reply:
updated